client/src/components/FormDisabled.js

```diff
@@ -0,0 +1,69 @@
+import { React, useState } from 'react';
+
+export default function FormDisabled(props) {
```

Comment:
What exactly is your `FormDisabled` component doing, it's very similar to AddVideo.js? Can you think of a way to reduce this duplication